tokio/tests/net_unix_pipe.rs

```diff
@@ -0,0 +1,429 @@
+#![cfg(feature = "full")]
+#![cfg(unix)]
+
+use tokio::io::{AsyncReadExt, AsyncWriteExt, Interest};
+use tokio::net::unix::pipe;
+use tokio_test::task;
```

Comment:
Can you please add the new types and the async methods to `tokio/tests/async_send_sync.rs`?

Reply:
Sure